supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -429,6 +429,18 @@ Ensure that you provide context about which type of quick start you are referrin
 
 *See also*:
 
+[discrete]
+[[red-hat-openshift-cluster-manager]]
+==== image:images/yes.png[yes] Red Hat OpenShift Cluster Manager (noun)
+*Description*: A managed service for Red Hat OpenShift that lets users create, subscribe, and manage different types of OpenShift clusters from a single user interface. After first mention, you can use OpenShift Cluster Manager. link:https://console.redhat.com/openshift[OpenShift Cluster Manager] is part of Red Hat Hybrid Cloud Console.
```

Comment:
* After first mention -> After the first mention
* Should there be a "the" in front of "Red Hat Hybrid Cloud Console"?

Reply:
I'm looking into your question of whether it should be "the Red Hat Hybrid Cloud Console"... will let you know.

Reply:
I talked to the M&A docs team and the consensus is that it should be 'the Red Hat Hybrid Cloud Console' - most pages, product and docs, use 'the'. I had seen a few docs mentions without 'the' and had thought that was correct -- we will be editing those in the future. Thanks Andrea!